aws/src/main/java/net/opentsdb/aura/aws/S3Uploader.java

```diff
@@ -31,9 +29,19 @@
 import software.amazon.awssdk.services.s3.model.PutObjectResponse;
 
 import java.io.ByteArrayInputStream;
+import java.time.temporal.ChronoUnit;
 
 public class S3Uploader implements Uploader {
```

Comment:
should this class be named as `MetaS3Uploader`?

Reply:
Probably eventually